services/base.js

```diff
@@ -34,6 +35,14 @@ class BaseService {
 
   // Metadata
 
+  /**
+   * When true, enable debugging on this service. Useful for troubleshooting
+   * a live server or in conjunction with `.only()` chained onto a service test.
+   */
+  static get debug() {
+    return false
```

Comment:
Would it be better if this was configured with an env var instead of hard-coded? Then we could set up scripts in `package.json` for `npm run start:quiet` or `npm run start:debug` or whatever (depending on whether we think the default should be on or off in dev) to toggle it on/off. That makes it easier to turn on/off, reduces the chance of code being committed with debug on and makes things easier if we're working on multiple services.

Alternatively it could be a command line flag. Combined with my other suggestion about log levels you could think of it as something like `npm start --verbosity 1`, `npm start --verbosity 2` etc for "errors only", "errors and warnings" and so on.. Maybe that level of detail is more than we need at this stage.

Reply:
Yea, that's a good idea. Better indeed than this approach of writing it into the class.

I think we probably don't need level of detail at this stage. Where I think that would come into play is if/when we start logging more detail on the server.